addons/medical/functions/fnc_useItem.sqf

```diff
@@ -24,13 +24,20 @@ if (isNil QGVAR(setting_allowSharedEquipment)) then {
     GVAR(setting_allowSharedEquipment) = true;
 };
 
-if (GVAR(setting_allowSharedEquipment) && {[_patient, _item] call EFUNC(common,hasItem)}) exitWith {
-    if (local _patient) then {
-        ["ace_useItem", [_patient, _item]] call CBA_fnc_localEvent;
-    } else {
-        ["ace_useItem", [_patient, _item], _patient] call CBA_fnc_targetEvent;
+private _use_patiten_item = {
+    params ["_patient", "_item"];
+    if ([_patient, _item] call EFUNC(common,hasItem)) exitWith {
+        if (local _patient) then {
+            ["ace_useItem", [_patient, _item]] call CBA_fnc_localEvent;
+        } else {
+            ["ace_useItem", [_patient, _item], _patient] call CBA_fnc_targetEvent;
+        };
+        [true, _patient];
```

Comment:
Generally we have the convention to leave away the `;` on return values don't we?
But this whole file isn't doing that.